docs/administering/ssl.md

```diff
@@ -2,71 +2,90 @@ The following is a process for self-hosted instances of Sandstorm to use SSL wit
 
 **Note**: Web browsers will display a big red certificate error when you try to connect to this install. This tutorial is appropriate if you are OK with reconfiguring web browsers to trust a custom certificate authority that you will create during this tutorial. For automatically-trusted SSL configuration, you will need to buy a certificate from a certificate authority.
 
-1. Make a copy openssl.cnf:
-  `cp /etc/ssl/openssl.cnf [directory_you_want_copy_to_be]`
+1. **Make a copy openssl.cnf:**
 
-2. Add the following to the end of the copied `openssl.cnf` file:
-  ```
-  [req]
-  req_extensions = v3_req
+        cp /etc/ssl/openssl.cnf [directory_you_want_copy_to_be]
 
-  [ v3_req ]
 
-  # Extensions to add to a certificate request
+2. **Add the following to the end of the copied `openssl.cnf` file:**
+        
+        [req]
+        req_extensions = v3_req
+        
+        [ v3_req ]
+        
+        # Extensions to add to a certificate request
+        
+        basicConstraints = CA:FALSE
+        keyUsage = nonRepudiation, digitalSignature, keyEncipherment
+        subjectAltName = @alt_names
+        
+        [alt_names]
+        DNS.1 = [your-domain-name].sandcats.io
+        DNS.2 = *.[your-domain-name].sandcats.io
 
-  basicConstraints = CA:FALSE
-  keyUsage = nonRepudiation, digitalSignature, keyEncipherment
-  subjectAltName = @alt_names
+3. **Create the Certificate Authority (CA) Key:**
 
-  [alt_names]
-  DNS.1 = [your-domain-name].sandcats.io
-  DNS.2 = *.[your-domain-name].sandcats.io
-  ```
-3. Create the Certificate Authority (CA) Key:
-  `openssl genrsa -out rootCA.key 4096`
+        openssl genrsa -out rootCA.key 4096
+    `rootCA.key` is the file for the Certificate Authority (CA) key.  
+    _  
```

Comment:
The underscores are there because I don't know how to do the line-spacing correctly with Markdown and mkdocs. I couldn't figure out how to get the spacing I wanted without adding something to a blank line and to keep the numbering correct. Maybe those with more experience with Markdown and mkdocs can figure out how to do the spacing right without the underscores.